cfn-resources/network-peering/cmd/resource/resource.go

```diff
@@ -26,6 +25,7 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"github.com/openlyinc/pointy"
```

Comment:
Should we use pointy or the aws one?

Reply:
Let's use the aws version for consistency.